workflow/scripts/filter_report_fuseq_wes.py

```diff
@@ -0,0 +1,158 @@
+
+import logging
+
+log = logging.getLogger()
+
+
+def get_breakpoints(breakpoint_file, sample):
+    fusion_breakpoint_dict = {}
+    next(breakpoint_file)
+    for fusion in breakpoint_file:
+        columns = fusion.strip().split("\t")
+        fusion_name = columns[18]
+        chrom1 = columns[3].split("__")[1]
+        break_point1_1 = int(columns[3].split("__")[2]) + int(columns[5])
+        break_point1_2 = break_point1_1 + int(columns[7])
+        break_point1 = f"{chrom1}:{break_point1_1}-{break_point1_2}"
+        chrom2 = columns[8].split("__")[1]
+        break_point2_1 = int(columns[8].split("__")[2]) + int(columns[10])
+        break_point2_2 = break_point2_1 + int(columns[12])
+        break_point2 = f"{chrom2}:{break_point2_1}-{break_point2_2}"
+        fusion_breakpoint_dict[fusion_name] = [break_point1, break_point2]
+    return fusion_breakpoint_dict
+
+
+def get_report_genes(gene_white_list):
+    report_genes = []
+    for gene in gene_white_list:
+        report_genes.append(gene.strip())
+    return report_genes
+
+
+def filter_fusion(sample, fusion_breakpoint_dict, report_genes, fusion_file, min_support, filter_on_fusiondb):
+    nr_report_genes = len(report_genes)
+    filtered_fusions = []
+    next(fusion_file)
+    for fusion in fusion_file:
+        columns = fusion.strip().split("\t")
+        fusion_name = columns[0]
+        gene1 = fusion_name.split("--")[0]
+        gene2 = fusion_name.split("--")[1]
+        reverse_fusion_name = f"{gene2}--{gene1}"
+        if nr_report_genes > 0 and not (gene1 in report_genes or gene2 in report_genes):
+            continue
+        support = int(columns[8])
+        if support < min_support:
+            continue
+        SR_support = int(columns[9])
+        MR_support = int(columns[10])
+        fusiondb = int(columns[11])
+        paralog = columns[12]
+        if fusiondb == 1 or filter_on_fusiondb is False:
+            break_points = ["", ""]
+            break_point1 = ""
+            break_point2 = ""
+            if SR_support > 0:
+                if fusion_name in fusion_breakpoint_dict:
+                    break_points = fusion_breakpoint_dict[fusion_name]
+                if reverse_fusion_name in fusion_breakpoint_dict:
+                    break_points = fusion_breakpoint_dict[reverse_fusion_name]
+                break_point1 = break_points[0]
+                break_point2 = break_points[1]
+            filtered_fusions.append([fusion_name, break_point1, "", break_point2, "", paralog, SR_support, MR_support, support])
```

Comment:
we would suggest that wi save the result as a dict, ex
```suggestion
            filtered_fusions.append({'name': fusion_name, 'break_point1': break_point1, 'exon1': "", 'break_point2': break_point2, 'exon2': "", 'paralog': paralog, 'SR_support': SR_support, 'MR_support, support': MR_support, support})
```
to make it  simpler to keep track of the content